### ISSUE TITLE
TypingDNA - Verify 2FA

### ISSUE THREAD

**Issue by pcd1000** [jazzband/django-two-factor-auth#482]
Are there any plans to integrate TypingDNA's - Verify 2FA into Django-two-factor-auth?

**Comment by claudep** [comment]
No, there are no such plans currently. The new plugin architecture progressively put in place in django-two-factor-auth should facilitate writing such plugins if anyone wants to contribute this functionality.

**Comment by pcd1000** [comment]
Many thanks claudep
